src/capi/impl/capi_sarray.cpp

```diff
@@ -723,11 +723,22 @@ EXPORT tc_sarray* tc_sarray_hash(const tc_sarray* sa, uint64_t salt, tc_error**
   ERROR_HANDLE_END(error, NULL);
 }
 
-EXPORT tc_sarray* tc_sarray_slice(const tc_sarray* sf, const int64_t start, const int64_t end, const int64_t stride, tc_error** error) {
+EXPORT tc_sarray* tc_sarray_slice(const tc_sarray* sf, const int64_t start, const int64_t slice, const int64_t end, tc_error** error) {
```

Comment:
In the header, this parameter is named `stride` -- here it's named `slice`. Should we be consistent between the two?